### PR TITLE
docs: add codesandbox to issue template for easier minimal reproduction of bugs

### PR DESCRIPTION
**Pull request by ChristianMurphy**
With many issues it can be difficult to get a clear understanding of what is broken, from a code sample.
Samples are often incomplete, and may not have all the context needed to replicate and understand the issue.
Codesandbox (or another sandbox tool) can help by providing a clean and shareable environment to demonstrate an issue.

### CODE REVIEW

**Review by vhf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1-bug.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Tell us how to reproduce this issue.  Please provide a working and simplified ex
 
 🎉 BONUS POINTS for creating a [minimal reproduction](https://stackoverflow.com/help/mcve) and uploading it to GitHub.  This will get you the fastest support.  🎉
 
+<!--
+Consider using a codesandbox to make the issue easy to replicate, here are two starters:
+https://codesandbox.io/s/remark-debug-ikwvx
+https://codesandbox.io/s/react-markdown-debug-9n4eg
```
Comment (wooorm) [inline .github/ISSUE_TEMPLATE/1-bug.md]:
```suggestion
remark only (for markdown to markdown): https://codesandbox.io/s/remark-debug-ikwvx
remark and rehype (for markdown to html): xxx
react-markdown: https://codesandbox.io/s/react-markdown-debug-9n4eg
```

**Review by wooorm — COMMENTED** [review]
Nice!

Added some text to clarify what the sandbox does, and what it uses.

Also: it would be good to steer folks to use unified -> remark-parse -> remark plugins -> remark-rehype -> rehype-raw when needed -> rehype plugins -> rehype-stringify.
No hard data for it but I’m feeling like a bunch can be better done at the rehype level

File: .github/ISSUE_TEMPLATE/1-bug.md (hunk 1 above)
Comment (wooorm) [reply .github/ISSUE_TEMPLATE/1-bug.md]: well that was fast: https://github.com/remarkjs/remark/discussions/559

Comment (ChristianMurphy) [reply .github/ISSUE_TEMPLATE/1-bug.md]: added in 8ab25ed

**Review by wooorm — APPROVED** [review]

**Pull request merged**